actions/update-deps/entrypoint.sh

```diff
@@ -26,6 +26,9 @@ git config --global --add safe.directory /github/workspace/main
 
 cd main
 
+# Additionally add Git submodules to the safe directory config
+git config --file .gitmodules --get-regexp path | awk '{ print $2 }' | xargs -I{} git config --global --add safe.directory {}
```

Comment:
the `eventing-istio` repo leverages those submodules. not sure if others do as well, but 

/lgtm
/approve